primitives/src/locktime/absolute.rs

```diff
@@ -91,12 +91,30 @@ impl LockTime {
     pub const SIZE: usize = 4; // Serialized length of a u32.
 
     /// Constructs a new `LockTime` from a prefixed hex string.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use bitcoin_primitives::absolute::LockTime;
+    /// let hex_str = "0x61cf9980"; // Unix timestamp for January 1, 2022
+    /// let lock_time = LockTime::from_hex(hex_str).expect("valid hex string");
```

Comment:
In 5f21b4726add5375994c2772d9705c560f55cfbd:

I really don't care about unwrapping in examples but just flagging that I saw this in case Tobin wants to change it.

Reply:
Yeah this is one of the items on the API guideline checklist.  I have replaced them with `?`.  Thanks.

Reply:
Yeah I personally also don't care but I think since we are going the full mile in the leaf crates we should just knuckle down and do it.

(Bother, I was told to use less slang/sayings but I struggle to express myself without these things coming to my mind.)

Reply:
This specific example could've just used numeric value.